src/js/game/LevelMVC/LevelView.js

```diff
@@ -1052,12 +1052,16 @@ module.exports = class LevelView {
         if (!this.controller.levelData.isEventLevel) {
           this.playPlayerAnimation("idle", playerPosition, facing, false, entity);
         }
-        this.playItemDropAnimation(destroyPosition, blockType, completionHandler);
+        if (completionHandler !== null) {
+          this.playItemDropAnimation(destroyPosition, blockType, completionHandler);
```

Comment:
I might be misunderstanding something about the completion handler, but don't we still want to play the animation?

Reply:
When calling the animation where I was, there was no completion handler, so it would die 3-4 functions deeper when it tries to call completion handler animations. 

This is something I'll run by Cole / Joel. I know for doors getting destroyed, we don't even HAVE miniblocks for them to drop, but for redstoneWire, we totally do. If we're invested in making sure those still drop, this shouldn't be TOO bad to fix.

Reply:
What I mean is, we should probably add the `if (completionHandler)` check at the lowest level; the level where it is actually getting called